workflow-trace-viewer/build.gradle.kts

```diff
@@ -0,0 +1,50 @@
+import org.jetbrains.compose.desktop.application.dsl.TargetFormat
+
+plugins {
+  id("kotlin-multiplatform")
+  alias(libs.plugins.jetbrains.compose)
+  alias(libs.plugins.compose.compiler)
+}
+
+kotlin {
+  jvm()
+
+  sourceSets {
+    jvmMain {
+      dependencies {
+        implementation(compose.runtime)
+        implementation(compose.foundation)
+        implementation(compose.material)
+        implementation(compose.ui)
+        implementation(compose.components.resources)
+        implementation(compose.components.uiToolingPreview)
+        implementation(libs.androidx.lifecycle.viewmodel.core)
+        implementation(libs.androidx.lifecycle.compose)
```

Comment:
I think a lot of these are forward looking what might be useful, right? But will these `androidx` ones be usable?

Reply:
Yeah, they're KMP now and are expected to used with Compose for Desktop.